airflow/providers/google/marketing_platform/hooks/search_ads.py

```diff
@@ -19,73 +19,154 @@
 
 from __future__ import annotations
 
-from typing import Any, Sequence
+from typing import Any
 
 from googleapiclient.discovery import build
 
 from airflow.providers.google.common.hooks.base_google import GoogleBaseHook
 
 
 class GoogleSearchAdsHook(GoogleBaseHook):
-    """Hook for Google Search Ads 360."""
+    """Hook for the Google Search Ads 360 Reporting API."""
 
     _conn: build | None = None
+    default_api_version: str = "v0"
 
     def __init__(
         self,
-        api_version: str = "v2",
+        api_version: str | None = None,
         gcp_conn_id: str = "google_cloud_default",
-        delegate_to: str | None = None,
-        impersonation_chain: str | Sequence[str] | None = None,
     ) -> None:
         super().__init__(
             gcp_conn_id=gcp_conn_id,
-            delegate_to=delegate_to,
-            impersonation_chain=impersonation_chain,
         )
-        self.api_version = api_version
+        self.api_version = api_version or self.default_api_version
 
-    def get_conn(self):
-        """Retrieve connection to Google SearchAds."""
+    def get_conn(self) -> Any:
```

Comment:
I think the return type here will be `Resource` from `googleapiclient.discovery` .